### PR TITLE
ABP-39-List Elements, Drag n Drop

### PR DESCRIPTION
**Pull request by amolra**
ABP-39-List Elements, Drag n Drop

### CODE REVIEW

File: src/app/shared/creative-editor/creative-editor.component.ts [hunk 1]
```diff
@@ -129,142 +107,118 @@ export class CreativeEditorComponent implements OnInit,AfterViewInit,OnDestroy
...
     this.renderer.appendChild(this.textStyle.nativeElement, newSector);
-    
   }
   private initializeEditor(): any {
     return grapesjs.init({
       container: '#gjs',
       autorender: true,
       forceClass: false,
-      // height: '100vh',
+      plugins: [myPlugin],
```
Comment (ajaypa-cybage) [inline src/app/shared/creative-editor/creative-editor.component.ts]: Please update the plugin name to be more meaningful

File: src/app/shared/plugins/popup-plugin.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import grapesjs from 'grapesjs';
+export default grapesjs.plugins.add('my-plugin-name', (editor, options) => {
```
Comment (ajaypa-cybage) [inline src/app/shared/plugins/popup-plugin.ts]: Please update the plugin name to be meaningful

File: src/app/shared/creative-editor/creative-editor.component.ts (hunk 1 above)
Comment (amolra) [reply src/app/shared/creative-editor/creative-editor.component.ts]: Done

File: src/app/shared/plugins/popup-plugin.ts (hunk 2 above)
Comment (amolra) [reply src/app/shared/plugins/popup-plugin.ts]: Done

**Comment by amolra** [comment]
this is out dated pull request so closing this.

**Pull request closed without merging**